gcc.spec

```diff
@@ -1,10 +1,10 @@
-### RPM external gcc 4.9.1
```

Comment:
Moving to gcc 4.9.2 requires building the bootstrap for `slc6` as well, as this branch is also used for other stuff (e.g. the CMSSW_7_5_DEVEL_X IBs, for root6).